tgui/packages/tgui/interfaces/CrewCredits/AntagonistsTab.tsx

```diff
@@ -0,0 +1,251 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author Mr. Moriarty (https://github.com/Mister-Moriarty)
+ * @license MIT
+*/
+
+import { Fragment } from 'inferno';
+import { useBackend } from '../../backend';
+import { Box, Collapsible, Divider, Icon, ItemList, LabeledList, Section, Stack } from '../../components';
+import { AntagonistStatisticsProps, AntagonistTabData, VerboseAntagonistProps } from './type';
+
+export const AntagonistsTab = (props, context) => {
+  const { data } = useBackend<AntagonistTabData>(context);
+
+  return (
+    <Box>
+      <Section>
+        <Stack
+          vertical
+          align="center"
+          my={3}>
+          <Stack.Item
+            mb={-2.5}
+            italic>
+            The Game Mode Was:
+          </Stack.Item>
+          <Stack.Item
+            fontSize={2.75}
+            bold>
+            {data.game_mode}
+          </Stack.Item>
+        </Stack>
+      </Section>
+      {data.verbose_antagonist_data?.map((antagonist, index) =>
+        (<Antagonist
+          key={index}
+          antagonist_roles={antagonist.antagonist_roles}
+          real_name={antagonist.real_name}
+          player={antagonist.player}
+          job_role={antagonist.job_role}
+          status={antagonist.status}
+          objectives={antagonist.objectives}
+          antagonist_statistics={antagonist.antagonist_statistics}
+          subordinate_antagonists={antagonist.subordinate_antagonists}
+        />)
```

Comment:
Not confirmed the data types match entirely, but you can probably just do:
```suggestion
        (<Antagonist
          key={index}
          {...antagonist}
        />)
```